packages/angular/projects/ui-angular/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aws-amplify/ui-angular",
-  "version": "3.2.18",
+  "version": "3.2.19",
```

Comment:
Think we need to scope this hotfix to just the React packages since Angular and Vue didn't see a MV when the React packages went from v4 to v5. `@aws-amplify/ui-angular@3.2.19` has already been published: https://www.npmjs.com/package/@aws-amplify/ui-angular/v/3.2.19

Reply:
Yeah I was just seeing that ui pkg v5.6.5 has already been published  https://www.npmjs.com/package/@aws-amplify/ui/v/5.6.5

Reply:
Going to try and see if pinning style dictionary in resolutions works for the v4 ui-react branch

Reply:
Going to close this one out and try a different strategy scoped down to ui-react v4 with resolutions in package.json for style-dictionary